std/range/package.d

```diff
@@ -4416,6 +4416,33 @@ body
     assert(approxEqual(rf, [0.0, 0.1, 0.2, 0.3, 0.4]));
 }
 
+///
+unittest
+{
+    import std.stdio;
+
+    /* The following groups all produce the same output of:
+     * 0 1 2 3 4
+     */
+
+    foreach (i; 0..5)
+        writef("%s ", i);
```

Comment:
Yah if unittests are supposed to write to the console we need to find a way to silently redirect them somewhere. It's unacceptable to have unittests emit random stuff like that.